### DISCUSSION TITLE
What is the expectations for detection under low light or evenings

### DISCUSSION THREAD

**Discussion by hua-bing** [blakeblackshear/frigate#15294] [General]
I have been running 0.12.1 with OpenVINO for more than one year.  Only recently I have realized that the detection in the evenings is almost close to non-existing for cameras that are not close to lights.  I have a set of 7 Wyze V2 and V3 cameras.  The detection works fine during day time (detection dimensions are 1280 x 720).  I checked the real time camera feeds for those cameras in the dark, and I did see moving objects.  Need to say that even those cameras are not under direct lighting, there are some low lights from the environment.  I.e., they are not in complete dark.  Is this expected?

**Comment by NickM-27** [comment]
The default models are trained on the COCO dataset which is not trained on camera images directly, so this is expected in general as they aren't trained for these low lighting scenarios. Using a purpose built model trained on images like these, like Frigate+, has shown to perform much better.

**Reply by hua-bing** [reply]
ok. Thanks @NickM-27
